include/mesh/coupled_interfaces/coupled_interfaces.hpp

```diff
@@ -12,6 +12,7 @@ namespace mesh {
  * @brief Information about coupled interfaces
  *
  */
+template <typename specfem::dimension::type DimensionType>
```

Comment:
Can you add a public constexpr static member variable for dimension to this class? i.e. `constexpr static auto dimension = DimensionType`

Reply:
Will do